src/appleseed.studio/mainwindow/project/entityeditor.cpp

```diff
@@ -37,9 +37,9 @@
 #include "utility/settingskeys.h"
 
 // appleseed.qtcommon headers.
+#include "utility/interop.h"
+#include "utility/miscellaneous.h"
```

Comment:
A big part of the PR could have been another *Move arround some scripts* PR

Reply:
I agree, but I shuffled things quite a bit before finding the proper solution, and by then it was deeply intertwined with the rest of the code...